dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -85,7 +88,7 @@ else
     cd /home/airflow/workspace
     ls
     cd e2e-benchmarking/workloads/$workload
-    export UUID=$(uuidgen | head -c16)-$AIRFLOW_CTX_TASK_ID-$(date '+%Y%m%d')
+    export UUID=$(uuidgen | head -c8)-$AIRFLOW_CTX_TASK_ID-$(date '+%Y%m%d')
```

Comment:
why are we reducing the length?

Reply:
Just to name benchmark resources shortly, I can revert it if it breaks something else.
ex. `kube-burner-cfg-499f9834-0526-49-hosted-2_node-density-20220210` to `kube-burner-cfg-499f9834-hosted-2_node-density-20220210`

Reply:
@whitleykeith @rsevilla87 hey do you guys think this would break something ? I am shortening UUID with 8 characters from `uuidgen`.

Reply:
this should be fine, as long as it's unique enough and 8 chars should definitely be enough (36^8 is like 2 trillion unique ids)